Gemfile

```diff
@@ -1,5 +1,30 @@
 source 'https://rubygems.org'
-source 'http://gems.dev.mas.local'
+source 'http://gems.dev.mas.local' do
+  gem 'action_plans', '~> 4.0.1'
+  gem 'advice_plans', '~> 3.1.0'
+  gem 'agreements', '~> 2.0.1'
+  gem 'baby_cost_calculator'
+  gem 'budget_planner', '~> 4.0.3'
+  gem 'car_cost_tool', '~> 1.0.4'
+  gem 'contribution_calculator', '~> 0.0.1'
+  gem 'cost_calculator_builder', '~> 0.1.10'
+  gem 'cutback_calculator', '~> 0.10'
+  gem 'debt_advice_locator', '~> 2.0'
+  gem 'debt_and_mental_health', '~> 1.0.1'
+  gem 'debt_free_day_calculator', '~> 2.1.0'
+  gem 'debt_test'
+  gem 'decision_trees', '~> 2.0.0'
+  gem 'feedback', '~> 0.3.1'
+  gem 'mortgage_calculator', '~> 1.3.5'
+  gem 'mas-assets'
+  gem 'meta-tags', github: 'moneyadviceservice/meta-tags'
+  gem 'payday_loans_intervention', '~> 1.4'
+  gem 'pensions_calculator', '~> 1.0.0'
+  gem 'quiz'
+  gem 'rio', '= 0.0.3', source: 'http://gems.dev.mas.local'
```

Comment:
Bundler complain about different sources, so I forced and the warnings were removed.